CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ ament_python_install_package(${PROJECT_NAME})
 # Install Python executables
 install(PROGRAMS
   karelics_vesc_can_driver/vesc_can_driver.py
+  karelics_vesc_can_driver/battery_status.py
```

Comment:
I know that I made this initial line, but since then I have learned that you should use: `catkin_install_python` also see: http://docs.ros.org/en/api/catkin/html/howto/format2/installing_python.html.  Since you are changing it, it might be a good time to update that.

Reply:
Also have a look at the best practices for installing nodes in the article I mentioned. Most notably:
- Names without `.py`
- Small scripts that only import and execute.

Reply:
Thanks, will check these out now!

Reply:
Related to installation, I believe that catkin way for installing isn't anymore preferred on ROS 2. But as with that approach, the proper way to install python executables would be by using setup.py. However, there is a bug related to --symlink-install when using that installation method (launch files and config files are not correctly symbolic linked), which is the reason why we are still using CMakeLists to install them.